### PR TITLE
Update our issue templates

### PR DESCRIPTION
**Pull request by blakewilson**
## Tasks

- [x] I have signed a [Contributor License Agreement (CLA)](https://github.com/wpengine/faustjs#contributor-license-agreement) with WP Engine.
- [ ] If a code change, I have written testing instructions that the whole team & outside contributors can understand.
- [ ] I have written and included a comprehensive [changeset](https://github.com/wpengine/faustjs/blob/canary/DEVELOPMENT.md#deployment) to properly document the changes I've made.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      label: Description
+      description: >-
+        Please write a brief description of the bug, including what you expected and what actually happened.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: >-
+        Please list the all the steps needed to reproduce the bug in an *isolated* way (e.g. a clonable GitHub repository, or a Faust.js snippet that works on a clean environment ).
```
Comment (mindctrl) [inline .github/ISSUE_TEMPLATE/bug_report.yml:23]:
```suggestion
        Please list all the steps needed to reproduce the bug in an *isolated* way (e.g. a clonable GitHub repository, or a Faust.js snippet that works on a clean environment ).
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: General Help Request
+    url: https://github.com/wpengine/faustjs/discussions
+    about: For general questions and help requests, create a new topic in Github Discussions
```
Comment (mindctrl) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: For general questions and help requests, create a new topic in GitHub Discussions
```

**Review by mindctrl — CHANGES_REQUESTED** [review]
Looks good! I left a couple minor suggestions.

**Comment by blakewilson** [comment]
@mindctrl Done! Re-review requested

**Review by mindctrl — APPROVED** [review]
🙌

**Pull request merged**